src/components/counters/_counters.scss

```diff
@@ -26,14 +26,62 @@ $counterSizeXS: componentSize(xs);
       top: 3px;
     }
 
-    &--red-60 {
-      background-color: $red-60;
+    &--black {
+      background-color: $black;
```

Comment:
let's use css variables here, so we're gradually making the dark mode easier.

this will be:
`background-color: var(--black)`

similarly to the other ones